addons/source-python/packages/source-python/listeners/tick.py

```diff
@@ -109,19 +109,17 @@ def add(self, delay):
 class Delay(WeakAutoUnload):
     """Execute a callback after a given delay."""
 
-    def __init__(
-        self, delay, callback, cancel_on_map_end=False, *args, **kwargs
-    ):
+    def __init__(self, delay, callback, cancel_on_map_end=False, args=(), kwargs={}):
```

Comment:
This is just one of many examples of searching for "python dangerous default value", which is what mutable types as argument defaults are called, returns.
http://stackoverflow.com/questions/9526465/best-practice-for-setting-the-default-value-of-a-parameter-thats-supposed-to-be